doc/classes/Dictionary.xml

```diff
@@ -118,63 +119,14 @@
 		};
 		[/csharp]
 		[/codeblocks]
-		[b]Note:[/b] Unlike [Array]s, you can't compare dictionaries directly:
-		[codeblocks]
-		[gdscript]
-		var array1 = [1, 2, 3]
-		var array2 = [1, 2, 3]
-
-		func compare_arrays():
-		    print(array1 == array2) # Will print true.
-
-		var dict1 = {"a": 1, "b": 2, "c": 3}
-		var dict2 = {"a": 1, "b": 2, "c": 3}
-
-		func compare_dictionaries():
-		    print(dict1 == dict2) # Will NOT print true.
-		[/gdscript]
-		[csharp]
-		// You have to use GD.Hash().
-
-		public Godot.Collections.Array array1 = new Godot.Collections.Array{1, 2, 3};
-		public Godot.Collections.Array array2 = new Godot.Collections.Array{1, 2, 3};
-
-		public void CompareArrays()
-		{
-		    GD.Print(array1 == array2); // Will print FALSE!!
-		    GD.Print(GD.Hash(array1) == GD.Hash(array2)); // Will print true.
-		}
-
-		public Godot.Collections.Dictionary dict1 = new Godot.Collections.Dictionary{{"a", 1}, {"b", 2}, {"c", 3}};
-		public Godot.Collections.Dictionary dict2 = new Godot.Collections.Dictionary{{"a", 1}, {"b", 2}, {"c", 3}};
-
-		public void CompareDictionaries()
-		{
-		    GD.Print(dict1 == dict2); // Will NOT print true.
-		}
-		[/csharp]
-		[/codeblocks]
-		You need to first calculate the dictionary's hash with [method hash] before you can compare them:
-		[codeblocks]
-		[gdscript]
-		var dict1 = {"a": 1, "b": 2, "c": 3}
-		var dict2 = {"a": 1, "b": 2, "c": 3}
-
-		func compare_dictionaries():
-		    print(dict1.hash() == dict2.hash()) # Will print true.
-		[/gdscript]
-		[csharp]
-		// You have to use GD.Hash().
-		public Godot.Collections.Dictionary dict1 = new Godot.Collections.Dictionary{{"a", 1}, {"b", 2}, {"c", 3}};
-		public Godot.Collections.Dictionary dict2 = new Godot.Collections.Dictionary{{"a", 1}, {"b", 2}, {"c", 3}};
-
-		public void CompareDictionaries()
-		{
-		    GD.Print(GD.Hash(dict1) == GD.Hash(dict2)); // Will print true.
-		}
-		[/csharp]
-		[/codeblocks]
-		[b]Note:[/b] When declaring a dictionary with [code]const[/code], the dictionary itself can still be mutated by defining the values of individual keys. Using [code]const[/code] will only prevent assigning the constant with another value after it was initialized.
+		The entries of a dictionary can be iterated with the [code]for[/code] keyword:
```

Comment:
From looking at the example it looks like GDScript iterates the keys of the dictionary, not the entries.